bloopgun/src/main/scala/bloop/bloopgun/Bloopgun.scala

```diff
@@ -362,9 +362,7 @@ class BloopgunCli(
       bloopVersion: String,
       logger: SnailgunLogger
   ): Option[mode.Out] = {
-    ServerStatus
-      .findServerToRun(bloopVersion, config.serverLocation, shell, logger)
```

Comment:
I can remove `findServerToRun` along with accompanying tests if we agree that this change is desirable, I wanted first to get some feedback